README.md

```diff
@@ -36,13 +36,15 @@ steps:
-|--------------------|----------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `cli_version`      | no       | /        | The version of the [steampunk-spotter]. If not provided, the latest version is used.                                                                                   |
-| `paths`            | no       | .        | List of paths to Ansible content files to be scanned. If not provided, the whole repository is scanned.                                                                |
-| `parse_values`     | no       | false    | If true, it parses values from Ansible task parameters and sends them for scanning.                                                                                    |                                
-| `display_level`    | no       | hint     | Displays check results with specified level or greater (e.g., warning will show all warnings and errors, but suppress hints). Available options: hint, warning, error. |
-| `ansible_version`  | no       | /        | Ansible version to use for scanning. If not provided, all Ansible versions are considered for scanning.                                                                |
+| Name              | Required | Default | Description                                                                                                                                                                        |
+|-------------------|----------|---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `cli_version`     | no       | /       | The version of the [steampunk-spotter]. If not specified, the latest version is used.                                                                                              |
+| `paths`           | no       | .       | List of paths to Ansible content files to be scanned. If not specified, the whole repository is scanned.                                                                           |
+| `project_id`      | no       | /       | ID of an existing target project in the app, where the scan result will be stored. If not specified, the first project of the user's first organization (in the app) will be used. |
+| `upload_values`   | no       | false   | Parses and uploads values from Ansible task parameters to the backend.                                                                                                             |                                
+| `upload_metadata` | no       | false   | Uploads metadata (i.e., file names, line and column numbers) to the backend.                                                                                                       |                                
```

Comment:
Shouldn't this default to true? Ince the main usecase is scanning public repos?

Reply:
To me `false` makes sense as the default choice, because the sending of any private data should be a deliberate opt in.